### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by chimwemwe007**
# Portfolio: add a contact form and finish the mobile version

- created a form for my partners to contribute
- add Name field accepting up to 30 chars.
- add Email field accepting only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- Linked formspree to my contact-form
- let my coding partners contribute to my contact form

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,34 +10,191 @@
...
+      </div>
+      <div class="it">
+        <h3>Skills</h3>
+        <hr>
+      </div>
+    </div>
+
+
+      <footer class="footer">
+        <section class="contact-form">
```
Comment (OLIPLICHE) [inline index.html:180]: - [ ] Please stick to the design templates in Figma as much as possible. ([See Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122))
- Your design is missing some ellipse and rectangular background: 
f1
- Original design
- 
f

File: index.html [hunk 2]
```diff
@@ -10,34 +10,191 @@
...
+          </p>
+  
+          <form action="https://formspree.io/f/mdobpqbd" method="post" class="form">
+                <input type="text" id="name" name="user_name" maxlength="30" placeholder="Chimwemwe Mkandawire"
+                  required />
+                <input type="email" id="mail" name="user_email" placeholder="chimwe.wire@gmail.com"
+                    required />
+                <textarea id="msg" name="user_message" maxlength="500" placeholder="Write your message here"
+                    required></textarea>
+                <button type="submit" class="btn">Get In Touch</button>
```
Comment (OLIPLICHE) [inline index.html:194]: - [ ] Please button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template: Kindly implement that on `Get In Touch`
- See Example from Figma.
-
b

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linter is green (the last commit have a green check mark (✅)) 👏
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Github flow is correctly used (Pull Request is made from the feature branch into the main branch) 👏
- Pull Request is descriptive 👏
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 3]
```diff
@@ -82,4 +65,230 @@ body {
...
+
+.card p {
+  padding: 0 20px;
+  font-size: 15px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 400;
+  color: #344563;
+}
+
+.btn {
```
Comment (Ismailco) [inline styles.css:171]: - [ ] As the previous reviewer suggests please change the colors of the buttons including button interactions such as hover, active (when it's pressed), disabled, and enable (default state) so they are styled like the Figma design

btn

File: index.html [hunk 4]
```diff
@@ -10,34 +10,192 @@
...
+      </div>
+      <div class="it">
+        <h3>Skills</h3>
+        <hr>
+      </div>
+    </div>
+
+
+      <footer class="footer">
+        <section class="contact-form">
```
Comment (Ismailco) [inline index.html:180]: - [ ] [optional] The form in your design is not centered, kindly make the form centered with the ``` Get in Touch``` button styled correctly and aligned to the left

File: index.html [hunk 5]
```diff
@@ -10,34 +10,192 @@
...
-          contact me.</p>
-
+  <section class="first-section">
+      <h1 class="about-me">
+        I'm Chimwemwe, Glad to see you!
+      </h1>
+      <p class="p1">I’m a software developer! I can help you build a product , feature or website Look through
+        some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+        contact me.</p>
         <p class="lets-connect"> LET'S CONNECT</p>
```
Comment (Ismailco) [inline index.html:31]: - [ ] [optional] ``` LET'S CONNECT``` should be a link with the color ``` #7F8CFF ```, please make the changes (you can use ```<a>``` tag instead) so your design can be as close as possible to the Figma one

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @chimwemwe007,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chimwemwe007** [comment]
Good evening Ismail,

Those are the changes I was told to make in the previous review, which I
made please double check.

Regards.

File: styles.css (hunk 3 above)
Comment (chimwemwe007) [reply styles.css:171]: @Ismailco I have already worked on the buttons I have added hove and active plus disabled what else do you want me to work on

Comment (Ismailco) [reply styles.css:171]: @chimwemwe007, Your buttons don't match the design, kindly fix the color of the buttons

- Your design:
Screenshot_20220507_213726Screenshot_20220507_213710Screenshot_20220507_213656

**Review by danielufeli — APPROVED** [review]
Hi @chimwemwe007 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

a

## Highlights
- Nice commits messages✔️
- Descriptive PR ✔️
- Good use of Github flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**